coalib/testing/LocalBearTestHelper.py

```diff
@@ -52,6 +54,29 @@ class LocalBearTestHelper(unittest.TestCase):
     If you miss some methods, get in contact with us, we'll be happy to help!
     """
 
+    def assertComparableObjectsEqual(self, observed_result, expected_result):
+        if len(observed_result) == len(expected_result):
+            messages = ''
+            for observed, expected in zip_longest(
+                    observed_result, expected_result):
```

Comment:
ah btw you don't need `zip_longest` any more, you can just `zip` :) we have ensured that the length is the same :)